build.sh

```diff
@@ -200,6 +217,7 @@ then
 		git submodule set-url Common/edk2-platforms                                 https://hub.nuaa.cf/tianocore/edk2-platforms.git
 		git submodule set-url Platform/EFI_Binaries                                 https://hub.nuaa.cf/edk2-porting/edk2-sdm845-binary.git
 		git submodule set-url GPLDrivers/Library/SimpleInit               			https://hub.nuaa.cf/BigfootACA/simple-init.git
+		git submodule set-url tools/Installer										https://hub.nuaa.cf/MollySophia/Flashable_UEFI_Installer.git
```

Comment:
please use spaces in the line to accommodate different tab lengths

Reply:
Fixed.
BTW it seems that hub.nuaa.cf is down?